src/System.Net.Http/src/System/Net/Http/SocketsHttpHandler/Http2Connection.cs

```diff
@@ -77,8 +76,14 @@ public Http2Connection(HttpConnectionPool pool, SslStream stream)
             _nextStream = 1;
             _initialWindowSize = DefaultInitialWindowSize;
             _maxConcurrentStreams = int.MaxValue;
+
+            _encodedAuthorityHostHeader = HPackEncoder.EncodeHeader(
```

Comment:
Any reason to not cache this on the pool instead of the connection?

Reply:
I thought I had a reason but now I can't think of it.  Will move it unless something proves problematic.

Reply:
Oh, right, my reason was I didn't want to add overhead to the HTTP/1.1 stuff.  But it should be relatively in the noise.